fastly/block_fastly_service_v1_logging_loggly.go

```diff
@@ -0,0 +1,202 @@
+package fastly
+
+import (
+	"fmt"
+	"log"
+
+	fst "github.com/fastly/go-fastly/fastly"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+var logglySchema = &schema.Schema{
+	Type:     schema.TypeSet,
+	Optional: true,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			// Required fields
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The unique name of the Loggly logging endpoint",
```

Comment:
The docs use the phrase "The name of the Loggly rule".

Reply:
That seems like a typo in the documentation to me.